codec/types/interface_registry.go

```diff
@@ -112,6 +112,10 @@ func (registry *interfaceRegistry) RegisterImplementations(iface interface{}, im
 }
 
 func (registry *interfaceRegistry) UnpackAny(any *Any, iface interface{}) error {
+	if any.TypeUrl == "" {
+		return nil
```

Comment:
nit, can you add a comment why are we returning here with `nil`?

Reply:
sure